table/tables/tables.go

```diff
@@ -299,6 +299,22 @@ func (t *TableCommon) FirstKey() kv.Key {
 	return t.RecordKey(math.MinInt64)
 }
 
+var tempBufStorePool = sync.Pool{
```

Comment:
Should we use `util/localpool/LocalPool` instead of `sync.Pool` since elements in it will be cleared during golang GC.

Reply:
The objects will be removed from `sync.Pool` is not used for two rounds of GC. I think this is good to reduce memory footprint. As `LocalPool` cannot grow and shrink the pool size, I think `sync.Pool` is better for now.